controllers/tmf-apis/ordering.js

```diff
@@ -606,6 +622,9 @@ var ordering = (function(){
             }
             // ELSE: If the user is the customer, order items don't have to be filtered
 
+            if (req.method.toUpperCase() === 'GET') {
+                ordering.note = sortByDate(ordering.note);
+            }
         });
 
         orderings = orderings.filter(function(ordering) {
```

Comment:
You have not modified the validateUpdate method, so providers cannot create notes since they are only authorized to modify items states. This happens for testing the interface using the same user as customer and seller...